sources/packages/web/src/services/StudentService.ts

```diff
@@ -58,14 +57,16 @@ export class StudentService {
     return studentContact;
   }
 
-  async getStudentConfirmInfo(): Promise<StudentConfirmInfo> {
-    const studentConfirmInfo = await ApiClient.Students.getStudentConfirmInfo();
-    //Formatting date received from api from 1998-03-24T00:00:00.000Z
-    //to March 24, 1998
-    studentConfirmInfo.birthDateFormatted = Helper.formatDate(
-      studentConfirmInfo.dateOfBirth,
-    );
-    return studentConfirmInfo;
+  async getStudentInfo() {
+    const studentInfo = await ApiClient.Students.getStudentInfo();
+    const studentInfoAll = {
+      ...studentInfo,
+      //Formatting date received from api from 1998-03-24T00:00:00.000Z to March 24, 1998
+      birthDateFormatted: Helper.formatDate(studentInfo.dateOfBirth),
```

Comment:
why two formatted birthDate? Can we keep date display formatting consistent? If require we can flag @roopfw

Reply:
For now, have done as per the figma. Can remove it when needed.